opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -161,9 +165,10 @@ def _python_path_without_directory(python_path, directory, path_separator):
 
 
 def is_instrumentation_enabled() -> bool:
```

Comment:
Add comment with the reason why checking both keys

Reply:
That could also be a question: should we support the old key?

Reply:
I think we shouldn't. Does anybody have thoughts about this?

Reply:
This [comment](https://github.com/open-telemetry/opentelemetry-python/issues/1737#issuecomment-815557962) explains the history of why we use a unique object instead of just the string. In terms of functionality, it would be nice to support plain text as well since it allows for easy use of this functionality from outside of the sdk (libraries won't have to explicitly reference `_SUPPRESS_HTTP_INSTRUMENTATION_KEY`) but it would defeat the purpose of having this context variable represented as a key. Might be a good discussion topic to bring up in the SIG

Reply:
@lzchen It's good to know the motivation behind that. I'll bring up this topic in the next SIG call.

Reply:
Discussed at SIG on 7/13 and we agree that we'll move forward supporting both old and new context keys.